### DISCUSSION TITLE
Is the STM32Ethernet library currently compatible with the STM32H563ZI?

### DISCUSSION THREAD

**Discussion by Insnomnia** [stm32duino/Arduino_Core_STM32#2222] [Libraries] [answered]
I saw there was a similar issue before for the STM32H7 derivative and I was wondering if the same applies here, where the library does't currently support this device. 
https://github.com/stm32duino/STM32Examples/issues/28
The same error messages also occur for my case.
Thanks for your help.

**Comment by fpistm** [comment, marked as answer]
Hi @Insnomnia 
While https://github.com/stm32duino/STM32Ethernet/issues/34 is not implemented it is not supported.
